packages/@lwc/template-compiler/src/scopeTokens.ts

```diff
@@ -46,12 +46,19 @@ export type scopeTokens = {
     cssScopeTokens: string[];
 };
 
+/**
+ * Generate the scope tokens for a given component. Note that this API is NOT stable and should be
+ * considered internal to the LWC framework.
+ * @param filename - full filename, e.g. `path/to/x/foo/foo.html`
+ * @param namespace - namespace, e.g. 'x' for `x/foo/foo.html`
+ * @param componentName - component name, e.g. 'foo' for `x/foo/foo.html`
+ */
```

Comment:
Added a big warning here that this is not stable. I can't imagine why someone would use it, but you never know... 🤷